target/linux/armsr/armv8/config-6.1

```diff
@@ -22,33 +51,58 @@ CONFIG_ARCH_WANTS_NO_INSTR=y
 CONFIG_ARCH_ZYNQMP=y
 CONFIG_ARM64=y
 CONFIG_ARM64_4K_PAGES=y
+CONFIG_ARM64_AMU_EXTN=y
+CONFIG_ARM64_BTI=y
 CONFIG_ARM64_CNP=y
 CONFIG_ARM64_CRYPTO=y
+CONFIG_ARM64_E0PD=y
 CONFIG_ARM64_EPAN=y
+CONFIG_ARM64_ERRATUM_1024718=y
 CONFIG_ARM64_ERRATUM_1165522=y
 CONFIG_ARM64_ERRATUM_1286807=y
+CONFIG_ARM64_ERRATUM_1319367=y
+CONFIG_ARM64_ERRATUM_1418040=y
+CONFIG_ARM64_ERRATUM_1463225=y
+CONFIG_ARM64_ERRATUM_1508412=y
+CONFIG_ARM64_ERRATUM_1530923=y
+CONFIG_ARM64_ERRATUM_1542419=y
+CONFIG_ARM64_ERRATUM_1742098=y
+CONFIG_ARM64_ERRATUM_2051678=y
+CONFIG_ARM64_ERRATUM_2054223=y
+CONFIG_ARM64_ERRATUM_2067961=y
+CONFIG_ARM64_ERRATUM_2077057=y
+CONFIG_ARM64_ERRATUM_2441007=y
+CONFIG_ARM64_ERRATUM_2441009=y
+CONFIG_ARM64_ERRATUM_2457168=y
+CONFIG_ARM64_ERRATUM_2658417=y
```

Comment:
Please activate these too:
```
ARM64_ERRATUM_2253138
ARM64_ERRATUM_2224489
ARM64_ERRATUM_2054223
ARM64_ERRATUM_2067961
```

Reply:
It looks like these are not in the default configuration:
````
hauke@hauke-t480:~/linux/linux-next-clean$ git grep ARM64_ERRATUM_2253138
Documentation/arm64/silicon-errata.rst:| ARM            | Neoverse-N2     | #2253138        | ARM64_ERRATUM_2253138       |
arch/arm64/Kconfig:config ARM64_ERRATUM_2253138
arch/arm64/kernel/cpu_errata.c:#ifdef CONFIG_ARM64_ERRATUM_2253138
hauke@hauke-t480:~/linux/linux-next-clean$ git grep ARM64_ERRATUM_2224489
Documentation/arm64/silicon-errata.rst:| ARM            | Cortex-A710     | #2224489        | ARM64_ERRATUM_2224489       |
Documentation/arm64/silicon-errata.rst:| ARM            | Cortex-X2       | #2224489        | ARM64_ERRATUM_2224489       |
arch/arm64/Kconfig:config ARM64_ERRATUM_2224489
arch/arm64/kernel/cpu_errata.c:#ifdef CONFIG_ARM64_ERRATUM_2224489
hauke@hauke-t480:~/linux/linux-next-clean$ git grep ARM64_ERRATUM_2054223
Documentation/arm64/silicon-errata.rst:| ARM            | Cortex-A710     | #2054223        | ARM64_ERRATUM_2054223       |
arch/arm64/Kconfig:config ARM64_ERRATUM_2054223
arch/arm64/kernel/cpu_errata.c:#ifdef CONFIG_ARM64_ERRATUM_2054223
hauke@hauke-t480:~/linux/linux-next-clean$ git grep ARM64_ERRATUM_2067961
Documentation/arm64/silicon-errata.rst:| ARM            | Neoverse-N2     | #2067961        | ARM64_ERRATUM_2067961       |
arch/arm64/Kconfig:config ARM64_ERRATUM_2067961
arch/arm64/kernel/cpu_errata.c:#ifdef CONFIG_ARM64_ERRATUM_2067961
hauke@hauke-t480:~/linux/linux-next-clean$ 
````
They were recently added, maybe they are not so important.

Reply:
> ARM64_ERRATUM_2253138
> ARM64_ERRATUM_2224489
> ARM64_ERRATUM_2054223
> ARM64_ERRATUM_2067961

These four are all relate to `CORESIGHT_TRBE` (CoreSight Trace Buffer extension) which does not get enabled on the default config or OpenWrt. On OpenWrt, CoreSight itself is disabled in the generic target config: https://github.com/openwrt/openwrt/blob/main/target/linux/generic/config-6.1#L1062